### ISSUE TITLE
Navigation of items with arrow keys

### ISSUE THREAD

**Issue by jnystromdesign** [Vuepic/vue-datepicker#11]
**Is your feature request related to a problem? Please describe.**
In an accessibility standpoint navigation of the days, month and years menu should be made with the arrow keys instead of using tabs, since it mimics a lot of the behavior patterns in native html date tags and makes it easier for disabled users to make a selection.  

**Describe the solution you'd like**
- When inside **day view menu** (the one that lets you navigate days in month), tabs *only* let you switch focus between 
   - selecting month, 
   - year, 
   - previous month
   - next month 
   - selecting **day selection**. 
   - If focused on **day selection**, arrows lets you navigate left-right-up-down, between items. Enter makes a selection.
- When inside **month view**, tabs *only* let you switch focus between selecting month and going back to calendar overview. If focused on month selection, arrows lets you navigate left-right-up-down, between items. Enter makes a selection and returns user to overview.
- When inside **year view**, tabs *only* let you switch focus between selecting month and going back to calendar overview. If focused on month selection, arrows lets you navigate left-right-up-down, between items. Enter makes a selection and returns user to overview.

**Illustrating tab stops and places where arrows should be used for navigation**
<img width="264" alt="days" src="https://user-images.githubusercontent.com/141703/161236949-08e9335f-a7cd-49d9-af2e-40d41386263d.png">
<img width="263" alt="month" src="https://user-images.githubusercontent.com/141703/161236964-e9485553-d8dd-40ca-98d8-b3ce429a2fac.png">
<img width="261" alt="years" src="https://user-images.githubusercontent.com/141703/161236984-4b00c8ec-1f66-4c5d-a84b-9dc28bc836a6.png">

**Comment by Jasenkoo** [comment]
I planned to implement this, since it is as you said, much easier to navigate, however, this will not be in the next version, but in the version after next, probably 3.2.
